config/hazelcast-client.yaml

```diff
@@ -21,4 +21,4 @@ hazelcast-client:
     hazelcast.logging.type: log4j2
   network:
     cluster-members:
-      - localhost:5801
```

Comment:
Hi, Why change port to 9000?

Reply:
> Hi, Why change port to 9000?

Can we keep using port 5801? Some of our users are already using seatunnel cluster, and modifying the port will have some impact.

Reply:
ok